src/test/java/com/hivemq/codec/decoder/MQTTMessageDecoderTest.java

```diff
@@ -174,4 +176,147 @@ public void test_second_connect_received() {
         assertFalse(channel.isOpen());
 
     }
+
+    @Test
+    public void test_connect_mqtt5_packet_size_too_large() {
```

Comment:
please name new tests according to the conventions

Reply:
this class is legacy, I'm just following the naming convection it already had.
See https://github.com/hivemq/hivemq-community-edition/pull/423/files/c7d4ce9e81b853c7dc35a6c09c7f795925f1016d#diff-11e37a0c7b609495e61a553c8ab4b5e851b84af733b30c4a26f70c05bede76b4L138

Should we change all the test names here?